tests/libvmi/storage.go

```diff
@@ -22,18 +22,23 @@ package libvmi
 import (
 	"fmt"
 
+	"kubevirt.io/kubevirt/tests/testsuite"
```

Comment:
I don't think that this is acceptable. `libvmi` should not depend on our testsuite. Ideally, it would become a general-purpose go binding for defining kubevirt VMI - not just in our tests but also elsewhere.